distribution/changelog.txt

```diff
@@ -45,6 +46,7 @@
 - Improved: [#16740] Allow staff patrol areas to be defined with individual tiles rather than groups of 4x4.
 - Improved: [#16764] [Plugin] Add hook ‘map.save’, called before the map is saved.
 - Improved: [#16925] The queue length of 1000 guests is lifted, and a warning for too long queues is added instead.
+- Improved: [#16999] The maximum price for the park entry has been raised to 999.
```

Comment:
Remove this one

Reply:
Why does it need to removed?

Reply:
Ah, I hadn't noticed the old one was still there too, my bad